code/game/machinery/alarm.dm

```diff
@@ -351,8 +353,15 @@ var/global/list/air_alarms = list()
 /obj/machinery/alarm/process()
 	if((stat & (NOPOWER|BROKEN|FORCEDISABLE)) || shorted || buildstage != 2)
```

Comment:
This makes the sound play even if the machinery is outright broken, manually disabled or under a stage of construction.

Reply:
You're no fun.